common/php/class-module.php

```diff
@@ -253,7 +253,7 @@ function enqueue_datepicker_resources() {
 	function get_current_post_type() {
 		global $post, $typenow, $pagenow, $current_screen;
 		//get_post() needs a variable
-		$post_int;
+		$post_int = '';
```

Comment:
Sure! Figured I'd let you fix it up if you wanted. As long as the notice goes away I'm happy :)